assumptions.tex

```diff
@@ -1,9 +1,11 @@
 The assumptions system allows users to specify that symbols have certain
 common mathematical properties, such as being positive, imaginary, or
 integer. SymPy is careful to never perform simplifications on an expression
-unless the assumptions allow them. For instance, the identity $\sqrt{t^2} = t$
-holds if $t$ is nonnegative ($t\ge 0$). However, for general complex $t$, no
-such identity holds.
+unless the assumptions allow them. For instance, the simplification $\sqrt{t^2}
+= t$ holds if $t$ is nonnegative ($t\ge 0$). However, for a complex $t$, the
+simplification is in general invalid, the correct identity that holds for all
```

Comment:
I moved it into a footnote in 17e83546e385b01559ccdff374fd431a4209eca8.

I would like it to be there, since the editor raised a question about it, so instead of "talk", let's provide an actual identity for `sqrt(x^2)` in terms of `x` that actually works for all complex `x`.